pz_server/api.py

```diff
@@ -430,9 +429,12 @@ def get_content(self, _id):
             dict: record data
         """
 
-        return self._download_request(f"{self._base_api_url}products/{_id}/content/")
-
-    def download_content(self, _id, save_in="."):
+        os.system("rm -rf .tmp")
```

Comment:
```suggestion
        os.system("rm -rf .tmp")
```
I suggest using the [tempfile](https://docs.python.org/3/library/tempfile.html):
```
# create a temporary directory using the context manager
with tempfile.TemporaryDirectory() as tmpdirname:
    print('created temporary directory', tmpdirname)
```